src/main/java/com/stripe/model/PaymentIntentNextSourceActionValueAuthorizeWithUrl.java

```diff
@@ -10,7 +10,7 @@
 @Getter
 @Setter
 @EqualsAndHashCode(callSuper = false)
-public class PaymentIntentSourceActionValueAuthorizeWithUrl extends StripeObject {
+public class PaymentIntentNextSourceActionValueAuthorizeWithUrl extends StripeObject {
```

Comment:
So this is tricky. We have a custom deserializer for that class. When you're renaming, you're breaking the custom deserializer. Is this expected? Is this part of autogen already too?

cc @ob-stripe who can explain this better than I would.

Reply:
Changed to `NextActionRedirectToUrl`